replication-api-impl/pom.xml

```diff
@@ -284,6 +284,7 @@
                         <Import-Package>
                             !org.jvnet.staxex,
                             ddf.security.permission,
+                            ddf.security.encryption;version="[1.0.0, 3.0.0]",
```

Comment:
❓ Why not `(2.13, 3)`?

Reply:
Because 2.13.6 and earlier export that bundle as version 1.0.0